manifests/bucketeer/charts/batch/templates/cronjob.yaml

```diff
@@ -48,11 +48,23 @@ spec:
               args:
                 - -c
                 - |
-                  echo "Start {{ .name }} job."
+                  echo '{"severity":"DEBUG","message":"Start {{ .name }} job."}'
```

Comment:
Since the start and success logs are for debug purposes only, I set them to report as DEBUG.